cmd/external-controller/main.go

```diff
@@ -83,7 +83,7 @@ func main() {
 		panic(err)
 	}
 	// Setup machine controller
-	machineActuator, err := external.NewMachineActuator(kubeClient, clusterClient, machineSetupConfigPath)
```

Comment:
Did the concept go away in other implementations or the Cluster API?

Reply:
Can you please explain what concept do you mean?
If you mean `machineSetupConfigPath`, I am pretty sure that we do not need it, for example, GCE provider uses it to configure VM instance, but we do not run any configuration at all, also if in the future we will see a need to add some common config, we can re-introduce it, but currently it makes things more complicated.
Also, I do not see any guidance on how to implement `actuator`, so it only our choice what components do we want to introduce.

Reply:
yes, but you can easily move all relevant stuff under machine object, so to have 3 different sources of configuration just make things more complex, and we do not really need it currently, that will create the additional barrier for peoples who want to join our effort.

Reply:
I had them there originally but that has the drawback of duplicating 95% of the IPMI config for each machine.  I plan to allow both. Please reverse the deletion.